README.md

```diff
@@ -465,9 +468,15 @@ From the [Oozie documentation](https://oozie.apache.org/docs/5.1.0/WorkflowFunct
 Currently the only supported way of configuring the map-reduce action is with the
 inline action configuration, i.e. using the `<configuration>` tag in the workflow's XML file definition.
 
+Issues in Github:
+* [Add support for config-default.xml](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/137)
+* [Add support for parameters section of the workflow.xml](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/138)
+* [Handle global configuration properties](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/134)
+
 **3. Streaming and pipes**
 
 Streaming and pipes are currently not supported.
+Issue in github [Implement streaming support](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/336)
```

Comment:
```suggestion

Issue in github [Implement streaming support](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/336)
```
This will start the sentence in a new line.